tf/env/staging/namespaces.tf

```diff
@@ -0,0 +1,18 @@
+resource "kubernetes_namespace" "api-job-namespace" {
+  metadata {
```

Comment:
This probably needs to have the wbaas-2 provider explicitly set like https://developer.hashicorp.com/terraform/language/providers/configuration#selecting-alternate-provider-configurations

e.g. `provider = kubernetes.wbaas-2`